models/cart-discount/src/cart-discount-custom-line-items-target/builders.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  createCompatibilityBuilder,
+  createSpecializedBuilder,
+  TModelFieldsConfig,
+} from '@commercetools-test-data/core';
+import { restFieldsConfig, graphqlFieldsConfig } from './fields-config';
+import type {
+  TCartDiscountCustomLineItemsTargetRest,
+  TCartDiscountCustomLineItemsTargetGraphql,
+} from './types';
+
+export const RestModelBuilder = () =>
+  createSpecializedBuilder({
+    name: 'CartDiscountCustomLineItemsTargetRestBuilder',
+    type: 'rest',
+    modelFieldsConfig: restFieldsConfig,
+  });
+
+export const GraphqlModelBuilder = () =>
+  createSpecializedBuilder({
+    name: 'CartDiscountCustomLineItemsTargetGraphqlBuilder',
+    type: 'graphql',
+    modelFieldsConfig: graphqlFieldsConfig,
+  });
+
+export const CompatModelBuilder = <
```

Comment:
The compatibility builders are only supposed to be used in current models we want to migrate to the new patterns so we should not add them to new ones.